iron-iconset-svg.html

```diff
@@ -196,7 +196,8 @@
       this._meta.key = this.name;
       this._meta.value = this;
 
-      this.async(function() {
+      // Give time for icons to be added
+      Polymer.RenderStatus.afterNextRender(this, () => {
         this.fire('iron-iconset-added', this, {node: window});
```

Comment:
Is this a 2.x legacy version? If so, maybe use this.dispatchEvent( ... );

Reply:
Polymer team's elements are hybrid and for the most part are using `this.fire`. I assume in the refactor of their elements they will switch to using `this.dispatchEvent(new CustomEvent())`